lib/Sema/CSApply.cpp

```diff
@@ -8120,6 +8117,7 @@ bool ConstraintSystem::applySolutionFix(Expr *expr,
       case CheckedCastKind::Coercion:
       case CheckedCastKind::BridgingCoercion:
         llvm_unreachable("Coercions handled in other disjunction branch");
+        return false;
```

Comment:
This isn't necessary (or useful). `llvm_unreachable` exits the process.

Reply:
I don't remember the circumstances for this. It will be removed.